site/index.html

```diff
@@ -76,6 +76,62 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>A new paradigm</h3>
+            <p>Express your ideas clearly and learn a new way of thinking about programming. Based on lambda calculus Haskell is a
+            purely functional programming language that features referential transparency, immutability and lazy evaluation.
+            Concepts that will blow your mind
+               &mdash; relearn programming while having an absolute blast.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Composition and predictability</h3>
+            <p>Reason about large pieces of code and compose them easily. There is no global state
+            or mutable variables obscuring the meaning of your program. The strong type system makes sure there are no surprises
+               &mdash; never again will you have to guess what your program does at execution time.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+           <h3>Declarative</h3>
+           <p>Write your programs declaratively by utilizing the power of pure functions and algebraic data types.
+              In Haskell we don't write how a program should be executed, we just describe its logic
+                 &mdash; never again be forced to think about evaluation order or execution details.
+           </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Performance</h3>
+            <p>Squeeze out the last ticks of your multi-core processors, thanks to best-in-class support for async,
+            concurrent and parallel programming... made possible via garbage collection and green threads.
+            Use advanced streaming libraries for ultra efficient data processing.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Abstraction</h3>
+            <p>Build powerful abstractions that are not possible in other languages. Only your imagination is the limit, due to
```

Comment:
"Only your imagination is the limit" feels a little vague. Why not just "Build powerful abstractions that are not possible in other languages, thanks to polymorphism...".

Rather than "has its roots in programming language research", why not: "has a design deeply rooted in programming language theory".

Reply:
I've been thinking to ditch this part and replace it with:

> Stability meets Evolution
> 
> Build powerful abstractions via latest functional programming principles. While Haskell’s core is rock stable, the language is constantly evolving, but does so through rigorous processes. Learn new tricks and features as you go, while relying on a solid foundation.

Which will then more nicely correlate with an alternative headline described here: https://discourse.haskell.org/t/emphasize-why-haskell-on-haskell-org-landing-page/12036/47

Reply:
Sounds good. Minor note: "via the latest functional programming principles". "rock solid" not "rock stable". I'd personally avoid "tricks" and replace it with "concepts".